### DISCUSSION TITLE
mountainduck doesn't open

### DISCUSSION THREAD

**Discussion by eeaapp** [iterate-ch/cyberduck#14916] [Q&A]
I downloaded mountainduck and the application opened at first with the ability to enter credentials then after exiting and trying to reopen (because I wasn't getting the correct username/passord) it no longer opened to anything so I deleted it and reinstalled it. Still not doing anything and I am trying to get the page back where I can open a connection. I have a MacBook Pro 2020/2021 (I don't remember).

**Comment by ylangisc** [comment]
Please refer to https://docs.cyberduck.io/mountainduck/interface/ on how the user interface works in Mountain Duck.
